tests/jenkins/TestRunNonSecurityPerfTestScript.groovy

```diff
@@ -149,6 +151,36 @@ class TestRunNonSecurityPerfTestScript extends BuildPipelineTest {
         ))
     }
 
+    @Test
+    void testRunSecurityPerfTestScript_verifyJob_aborted() {
+        binding.setVariable('BUNDLE_MANIFEST', 'tests/jenkins/data/opensearch-1.3.0-bundle.yml')
+        binding.setVariable('HAS_SECURITY', true)
+        helper.registerAllowedMethod('sh', [String.class], { String cmd ->
+            updateBuildStatus('ABORTED')
+        })
+        runScript("jenkins/opensearch/perf-test.jenkinsfile")
+        
+        assertJobStatusAborted() 
+        assertCallStack()
+        assertCallStack().contains("perf-test.cfnDescribe({stack=test-single-security-1236-x64-perf-test})")
+        assertCallStack().contains("'test-single-security-1236-x64-perf-test' does not exist, nothing to remove")
```

Comment:
Please test one of the scenario with deletion. You can do that by setting the if condition to true.

Reply:
makes sense, we don't need to assert the same thing in both cases ...  I have modified one of them to assert the cfnDelete .. sorry to ask you for another review .. thanks !!